src/include/sof/lib_manager.h

```diff
@@ -127,13 +127,12 @@ void lib_manager_init(void);
 /*
  * \brief Register module on driver list
  *
- * param[in] desc - library manifest descriptor
- * param[in] module_id - used to get manifest offset for module
+ * param[in] component_id - component id
  *
  * Creates new comp_driver_info and initialize it for module from library
  * Adds new module to sof_get()->comp_drivers list
  */
-int lib_manager_register_module(struct sof_man_fw_desc *desc, int module_id);
+int lib_manager_register_module(const uint32_t component_id);
```

Comment:
why is this a component ID?

Reply:
I found the name `module_id` to be ambiguous. I wanted to distinguish the "full" id coming from ipc (called `component_id` which contains instance id, lib id and module id) from plain `module_id = IPC4_MOD_ID(ipc_config->id);`.

Reply:
We could explain this in the doxygen description for this API ?